src/DataFramesMeta.jl

```diff
@@ -43,27 +46,39 @@ replace_syms!(e::Expr, membernames) =
 
 protect_replace_syms!(e, membernames) = e
 protect_replace_syms!(e::Expr, membernames) =
-    e.head == :quote ? e : replace_syms!(e, membernames)
+    if e.head == :quote
```

Comment:
The old version as a one-liner was more readable IMHO. If you change that you should use the long `function` syntax, but that doesn't sound like a good idea.